web/themes/custom/contribtracker/package.json

```diff
@@ -20,6 +22,14 @@
     "breakpoint-sass": "^3.0.0",
     "browser-sync": "^3.0.2",
     "cypress": "13.12.0",
+    "cypress-grep": "^2.0.1",
+    "cypress-xpath": "^2.0.1",
```

Comment:
This package is not supported anymore. It seems very suspicious. https://www.npmjs.com/package/cypress-xpath

Reply:
Let me update this one. Thanks for the catch!